fire/core.py

```diff
@@ -684,6 +685,7 @@ def _ParseKeywordArgs(args, fn_args, fn_keywords):
     remaining_args: A list of the unused arguments from the original args.
```

Comment:
- Please update the Returns: section of the docstring.
- Also how about "remaining_kwargs" instead of unconsumed_named_args for consistency with the existing terminology?